src/test/java/ua/mateacademy/tania/lesson10/FruitShopMockSpyTest.java

```diff
@@ -0,0 +1,125 @@
+package ua.mateacademy.tania.lesson10;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.time.LocalDate;
+import java.time.Month;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+import static ua.mateacademy.tania.lesson10.FruitType.*;
+
+/**
+ * Created by Tania Nebesna on 23.01.2019.
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class FruitShopMockSpyTest {
+    @Spy
+    @InjectMocks
+    private FruitShop fruitShop;
+
+    @Mock
+    private Fruit strawberry;
+    @Mock
+    private Fruit apple;
+    @Mock
+    private Fruit pear;
+    @Mock
+    private Fruit orange;
+
+    @Before
```

Comment:
Shouldn't it be @BeforeClass here? Why or why not?

Reply:
to use @BeforeClass my fields must be static
and with static and mock should be carefull that`s why I use non-static fields and @Before